res/asset_packer/src/asset_packer.c

```diff
@@ -439,53 +439,61 @@ static void pack_group(int group_id)
     snprintf(current_dir, FILE_NAME_MAX, "%s/%s/", PACKED_ASSETS_DIR, group->name);
     snprintf(current_file, FILE_NAME_MAX, "%s/%s", PACKED_ASSETS_DIR, group->path);
 
-    FILE *xml_dest = fopen(current_file, "wb");
-
-    if (!xml_dest) {
-        log_error("Failed to create file", group->path, 0);
-        return;
-    }
-
-    fprintf(xml_dest, "<?xml version=\"1.0\"?>\n");
-    fprintf(xml_dest, "<!DOCTYPE assetlist>\n\n");
-
-    fprintf(xml_dest, "<!-- XML auto packed by asset_packer. DO NOT use as a reference.\n");
-    fprintf(xml_dest, "     Use the assets directory from the source code instead. -->\n\n");
-
-    fprintf(xml_dest, "<assetlist name=\"%s\">%s", group->name, FORMAT_NEWLINE);
+    buffer buf;
+    int buf_size = 5 * 1024 * 1024;
```

Comment:
I'd much rather have a dynamic size here, but that's something to think about in the future.